frontend/src/components/BlockDetails/BlockDetailsContainer.vue

```diff
@@ -36,15 +36,12 @@ const paginationVars = {
 }
 
 type Props = {
-	id?: string
 	hash?: string
```

Comment:
We need the hash be present now, so it should not be optional
```suggestion
	hash: string
```